packages/cli/src/utils/convert.js

```diff
@@ -40,8 +40,10 @@ const createFile = async (content, filename, dir) => {
   endSpinner();
 };
 
-const prettifyJs = (code) =>
-  prettier.format(code, { singleQuote: true, parser: 'babel' });
+const prettifyJs = async (code) => {
+  return prettier.format(code, { singleQuote: true, parser: 'babel' });
```

Comment:
`prettier.format` returns a Promise since v3. This means `renderDefinitionSlice` below which uses `prettifyJs` needs to `await` it. Interestingly, the callers for `renderDefinitionSlice` itself already awaited the response 👍🏻